### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by LHJE**
Closes User Registration Milestone

**Contributors**
Leah, Luke, Priya, Grant

**What this did**
These stories allowed us to register a user on our site.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,6 +12,8 @@
     <nav class = "topnav">
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
+      <%= link_to "Register", "/register"%>
+
```
Comment (leahriffell) [inline app/views/layouts/application.html.erb:16]: can you please remove this extra line?

File: config/routes.rb [hunk 2]
```diff
@@ -33,4 +33,9 @@
   get "/orders/new", to: "orders#new"
   post "/orders", to: "orders#create"
   get "/orders/:id", to: "orders#show"
+
+  get "/register", to: "users#new"
+  resources :users, only: [:create]
```
Comment (leahriffell) [inline config/routes.rb:38]: Could we maybe move resources up one line to sort of kick off this block of user routes?

**Review by leahriffell — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (LHJE) [reply app/views/layouts/application.html.erb]: Got it!  Great catch :)

File: config/routes.rb (hunk 2 above)
Comment (LHJE) [reply config/routes.rb:37]: It's done!  Nicely done.

**Review by leahriffell — APPROVED** [review]

**Pull request merged**